src/components/FloatingHelper/HelperContent/HelperContent.st.css

```diff
@@ -9,3 +13,12 @@
 .action {
   margin-top: 18px;
 }
+
+.image {
+  margin-left: 18px;
```

Comment:
The fact that you placed flex on the image demonstrates that the image you used (for example int the story) is not aligned, so actually, the icon is not good enough if you need ti flex it.